doc/ref_model/chapters/chapter01.rst

```diff
@@ -0,0 +1,99 @@
+
+Scope
+-----
+
+This **Reference Model** document focuses on identifying the abstractions, and associated concepts, that are needed to represent the cloud infrastructure. **Figure 1-1** below highlights its scope in more details.
+
+.. image:: ../figures/ch01_scope.png
+   :alt: "Figure 1-1: Scope of Reference Model"
+
+**Figure 1-1:** Scope of Reference Model
+
+This document specifies:
+
+-  **Cloud Infrastructure abstraction**: in context with how it interacts with the other components required to build a complete cloud system that supports workloads deployed in Virtual Machines (VM) or containers. Network function workloads that are deployed on virtual machines and containers are referred to as virtual network functions (VNF) and containerised network functions (CNF), respectively; please note that it is now more common to refer CNFs as cloud native network functions.
+
+   -  **Cloud Infrastructure capabilities & metrics**: A set of cloud infrastructure capabilities and metrics required to perform telco scale network functions and satisfy their performance criterion.
+   -  **Infrastructure profiles catalogue**: A catalogue of standard infrastructure software and hardware configurations, referred to as profiles; these profiles abstract the infrastructure for the workloads. Only a few profiles, with well-defined characteristics, can meet the operational and performance requirements of all workloads.
+
+-  **Cloud Infrastructure Software and Hardware profiles**:
+
+   -  **Cloud Infrastructure software profiles**: These software profiles are components of the corresponding infrastructure profiles within the infrastructure profiles catalogue, and specify the host infrastructure software configurations.
+   -  **Cloud Infrastructure hardware profiles**: These hardware profiles are components of the corresponding infrastructure profiles within the infrastructure profiles catalogue, and specify the host infrastructure hardware configurations.
+
+-  **Conformance and verification**:
+
+   -  **Conformance programs**: These define the requirements for verification and validation programs for both the cloud infrastructure and workloads.
+   -  **Test framework**: This document provides input into test suites to allow conformance of cloud infrastructure and workloads.
+
+Principles
+----------
+
+The Reference Model specifications conform to the overall principles defined in `Anuket Principles <../../common/chapter00.md#2.0>`__.
+
+Definitions/Terminology/Abbreviations
+-------------------------------------
+
+To help guide the reader, the Reference Model `Glossary <../../common/glossary.md>`__ provides an introduction to the main terms used within this document and throughout the project in general. These definitions are, with a few exceptions, based on the ETSI GR NFV 003 [1] definitions. In a few cases, they have been modified to avoid deployment technology dependencies only when it seems necessary to avoid confusion.
```

Comment:
```suggestion
To help guide the reader, the Reference Model `Glossary <../../common/glossary>`__ provides an introduction to the main terms used within this document and throughout the project in general. These definitions are, with a few exceptions, based on the ETSI GR NFV 003 [1] definitions. In a few cases, they have been modified to avoid deployment technology dependencies only when it seems necessary to avoid confusion.
```

Reply:
See https://github.com/cntt-n/CNTT/pull/2746/files#r776460947